### DISCUSSION TITLE
Polylookup returning blanks in dropdown

### DISCUSSION THREAD

**Discussion by bhavk26** [khoait/DCE.PCF#113] [General] [answered]
Fetching multiple records from lookup table but as I add the custom control it shows blank values in dropdown for simple nton.

**Comment by khoait** [comment]
Hi @bhavk26. Could you share your configuration for the component? Could be relationship name configured incorrectly.

**Reply by bhavk26** [reply]
Relationship name is configured correctly
![Screenshot 2024-05-28 221923](https://github.com/khoait/DCE.PCF/assets/123792424/1d9d007e-cf1e-4fbf-920d-cfb35fada05d)

**Reply by bhavk26** [reply]
[dummy_1_0_0_1_managed.zip](https://github.com/khoait/DCE.PCF/files/15473315/dummy_1_0_0_1_managed.zip)

**Reply by khoait** [reply]
Thanks! i'll look into this and get back to you.

**Reply by bhavk26** [reply]
Sure, Thank you very much

**Reply by khoait** [reply, marked as answer]
Hi @bhavk26, found the issue. the relationship name should include the prefix of your solution. this should work `new_cr0a8_newtable1_cr0a8_table_cr0a8_table`

please see wiki here: https://github.com/khoait/DCE.PCF/wiki/PolyLookup#sample-relationship-names

**Reply by bhavk26** [reply]
Thanks for taking time,  let me check that

**Reply by bhavk26** [reply]
@khoait great it worked , one last thing can you remove the id and etn fields from generated json as it doesn't look tidy from frontend
